### PR TITLE
remove .cif/.pdb in filenames & remove _MODEL_ # in .lookup

### PR DESCRIPTION
**Pull request by ChaSooyoung**

### CODE REVIEW

File: src/strucclustutils/structcreatedb.cpp [hunk 1]
```diff
@@ -74,6 +74,17 @@ static inline bool compareByFirst(const std::pair<T, U>& a, const std::pair<T, U
     return a.first < b.first;
 }
 
+std::string removeMODEL(const std::string& input) {
+    size_t modelIndex = input.find("MODEL");
+    if (modelIndex == std::string::npos)
```
Comment (milot-mirdita) [inline src/strucclustutils/structcreatedb.cpp:79]: Please don't omit curly brackets `{}` for single line `if` statements.

File: src/strucclustutils/structcreatedb.cpp [hunk 2]
```diff
@@ -74,6 +74,17 @@ static inline bool compareByFirst(const std::pair<T, U>& a, const std::pair<T, U
     return a.first < b.first;
 }
 
+std::string removeMODEL(const std::string& input) {
```
Comment (milot-mirdita) [inline src/strucclustutils/structcreatedb.cpp:77]: Please use camelCaseing.

File: src/strucclustutils/structcreatedb.cpp [hunk 3]
```diff
@@ -853,8 +864,9 @@ int structcreatedb(int argc, const char **argv, const Command& command) {
         for (unsigned int id = 0; id < readerHeader.getSize(); id++) {
             char *header = readerHeader.getData(id, 0);
             entry.id = readerHeader.getDbKey(id);
-            entry.entryName = Util::parseFastaHeader(header);
-            std::pair<size_t, unsigned int> fileIdModelEntry = entrynameToFileId[entry.entryName];
+            std::string entryNamewithMODEL = Util::parseFastaHeader(header);
```
Comment (milot-mirdita) [inline src/strucclustutils/structcreatedb.cpp:867]: Please camelCase this too `entryNameWithModel`

File: src/strucclustutils/structcreatedb.cpp (hunk 1 above)
Comment (ChaSooyoung) [reply src/strucclustutils/structcreatedb.cpp]: Done! thanks

**Pull request merged**